gravitee-apim-plugin/gravitee-apim-plugin-entrypoint/gravitee-apim-plugin-entrypoint-webhook/src/test/java/io/gravitee/plugin/entrypoint/webhook/WebhookEntrypointConnectorFactoryTest.java

```diff
@@ -87,4 +93,58 @@ void shouldCreateConnectorWithNullConfiguration() {
         assertThat(connector).isNotNull();
         assertThat(connector.configuration).isNotNull();
     }
+
+    @Test
+    void shouldCreateConnectorWithStaticProxyConfiguration() throws Exception {
+        when(deploymentContext.getTemplateEngine()).thenReturn(TemplateEngine.templateEngine());
+
+        try (final var in = this.getClass().getResourceAsStream("/config/static-proxy.json")) {
+            WebhookEntrypointConnector connector = webhookEntrypointConnectorFactory.createConnector(
+                deploymentContext,
+                Qos.NONE,
+                new String(in.readAllBytes())
+            );
+            assertThat(connector).isNotNull();
+            assertThat(connector.configuration).isNotNull();
+
+            final var config = connector.configuration;
+            assertThat(config).isNotNull();
+            assertThat(config.getProxyOptions()).isNotNull();
+            assertThat(config.getProxyOptions().getHost()).isNotNull();
+            assertThat(config.getProxyOptions().getHost()).isEqualTo("proxy.host");
```

Comment:
I'm not sure to understand why we split in 2 asserts. AssertJ has a fluent API

This should work 
```suggestion
            assertThat(config.getProxyOptions().getHost()).isNotNull().isEqualTo("proxy.host");
```